lib/ra10ke.rb

```diff
@@ -61,7 +61,7 @@ def initialize(*args)
                     nil
                   end
                 end.select { |tag| !tag.nil? }.sort.last.to_s.downcase
-                latest_ref = tags.detect { |tag| tag.downcase == latest_tag || tag.downcase == latest_tag[1..-1] }
+                latest_ref = tags.detect { |tag| tag.downcase == latest_tag || (latest_tag[0] == 'v' && tag.downcase == latest_tag[1..-1]) }
```

Comment:
Wouldn't it be better to do `tags.detect { |tag| tag.downcase == latest_tag || "v#{tag.downcase}" == latest_tag }` or even `tags.detect { |tag| [tag.downcase, "v#{tag.downcase}"].include?(latest_tag) }`

Reply:
Indeed, makes sense.